### ISSUE TITLE
SQLiteCache is not defined

### ISSUE THREAD

**Issue by katchy3132** [pictuga/morss#113]
with the latest docker image update getting this error:

> NameError: name 'SQLiteCache' is not defined
> morss  | /usr/bin/python
> morss  | run
> morss  | + command -v python
> morss  | + '[' -z run ]
> morss  | + '[' run '=' sh ]
> morss  | + '[' run '=' bash ]
> morss  | + command -v run
> morss  | + run
> morss  | + gunicorn --bind 0.0.0.0:8000 --preload --access-logfile - morss
> morss  | Traceback (most recent call last):
> morss  |   File "/usr/bin/gunicorn", line 8, in <module>
> morss  |     sys.exit(run())
> morss  |              ^^^^^
> morss  |   File "/usr/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 67, in run
> morss  |     WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
> morss  |   File "/usr/lib/python3.11/site-packages/gunicorn/app/base.py", line 231, in run
> morss  |     super().run()
> morss  |   File "/usr/lib/python3.11/site-packages/gunicorn/app/base.py", line 72, in run
> morss  |     Arbiter(self).run()
> morss  |     ^^^^^^^^^^^^^
> morss  |   File "/usr/lib/python3.11/site-packages/gunicorn/arbiter.py", line 58, in __init__
> morss  |     self.setup(app)
> morss  |   File "/usr/lib/python3.11/site-packages/gunicorn/arbiter.py", line 118, in setup
> morss  |     self.app.wsgi()
> morss  |   File "/usr/lib/python3.11/site-packages/gunicorn/app/base.py", line 67, in wsgi
> morss  |     self.callable = self.load()
> morss  |                     ^^^^^^^^^^^
> morss  |   File "/usr/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
> morss  |     return self.load_wsgiapp()
> morss  |            ^^^^^^^^^^^^^^^^^^^
> morss  |   File "/usr/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
> morss  |     return util.import_app(self.app_uri)
> morss  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> morss  |   File "/usr/lib/python3.11/site-packages/gunicorn/util.py", line 359, in import_app
> morss  |     mod = importlib.import_module(module)
> morss  |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> morss  |   File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
> morss  |     return _bootstrap._gcd_import(name[level:], package, level)
> morss  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> morss  |   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
> morss  |   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
> morss  |   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
> morss  |   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
> morss  |   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
> morss  |   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
> morss  |   File "/usr/lib/python3.11/site-packages/morss/__init__.py", line 22, in <module>
> morss  |     from .morss import *
> morss  |   File "/usr/lib/python3.11/site-packages/morss/morss.py", line 29, in <module>
> morss  |     from . import caching, crawler, feeds, readabilite
> morss  |   File "/usr/lib/python3.11/site-packages/morss/caching.py", line 116, in <module>
> morss  |     default_cache = SQLiteCache(
> morss  |                     ^^^^^^^^^^^
> morss  | NameError: name 'SQLiteCache' is not defined
> morss exited with code 1

I have the follwing defined in my docker compose file : 

 ```
            - CACHE=sqlite
            - SQLITE_PATH=/tmp/morss.cache
            - CACHE_SIZE=1000
            - CACHE_LIFESPAN=4500
        volumes:
            - ${PWD}/morss.cache:/tmp/morss.cache
```

**Comment by pictuga** [comment]
SQLite cache has been phased out in favour of diskcache. There is still some legacy SQLite code that needs to be cleaned out (thus I'll leave this error open).

**Comment by pictuga** [comment]
SQLite was phased out some time ago but the ci/cd has been broken for a while and only fixed a few days ago, hence the breakage

**Comment by pictuga** [comment]
Should be fixed in 6529fdb
